<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>459</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabMain">
          <widget class="TabSudAuswahl" name="tabSudAuswahl">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabsudauswahl.svg</normaloff>
             <disabledoff>:/images/disabled/tabsudauswahl.svg</disabledoff>:/images/light/tabsudauswahl.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Sudauswahl</string>
           </attribute>
          </widget>
          <widget class="TabRezept" name="tabRezept">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabrezept.svg</normaloff>
             <disabledoff>:/images/disabled/tabrezept.svg</disabledoff>:/images/light/tabrezept.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Rezept</string>
           </attribute>
          </widget>
          <widget class="TabBraudaten" name="tabBraudaten">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabbraudaten.svg</normaloff>
             <disabledoff>:/images/disabled/tabbraudaten.svg</disabledoff>:/images/light/tabbraudaten.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Braudaten</string>
           </attribute>
          </widget>
          <widget class="TabAbfuellen" name="tabAbfuelldaten">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tababfuellen.svg</normaloff>
             <disabledoff>:/images/disabled/tababfuellen.svg</disabledoff>:/images/light/tababfuellen.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Abfülldaten</string>
           </attribute>
          </widget>
          <widget class="TabGaerverlauf" name="tabGaerverlauf">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabgaerverlauf.svg</normaloff>
             <disabledoff>:/images/disabled/tabgaerverlauf.svg</disabledoff>:/images/light/tabgaerverlauf.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Gärverlauf</string>
           </attribute>
          </widget>
          <widget class="TabZusammenfassung" name="tabZusammenfassung">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabzusammenfassung.svg</normaloff>
             <disabledoff>:/images/disabled/tabzusammenfassung.svg</disabledoff>:/images/light/tabzusammenfassung.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Zusammenfassung</string>
           </attribute>
          </widget>
          <widget class="TabEtikette" name="tabEtikette">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabetikette.svg</normaloff>
             <disabledoff>:/images/disabled/tabetikette.svg</disabledoff>:/images/light/tabetikette.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Etikette</string>
           </attribute>
          </widget>
          <widget class="TabBewertung" name="tabBewertung">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabbewertung.svg</normaloff>
             <disabledoff>:/images/disabled/tabbewertung.svg</disabledoff>:/images/light/tabbewertung.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Bewertung</string>
           </attribute>
          </widget>
          <widget class="TabBrauUebersicht" name="tabBrauuebersicht">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabbrauuebersicht.svg</normaloff>
             <disabledoff>:/images/disabled/tabbrauuebersicht.svg</disabledoff>:/images/light/tabbrauuebersicht.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Brauübersicht</string>
           </attribute>
          </widget>
          <widget class="TabAusruestung" name="tabAusruestung">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabausruestung.svg</normaloff>
             <disabledoff>:/images/disabled/tabausruestung.svg</disabledoff>:/images/light/tabausruestung.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Ausrüstung</string>
           </attribute>
          </widget>
          <widget class="TabRohstoffe" name="tabRohstoffe">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabrohstoffe.svg</normaloff>
             <disabledoff>:/images/disabled/tabrohstoffe.svg</disabledoff>:/images/light/tabrohstoffe.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Rohstoffe</string>
           </attribute>
          </widget>
          <widget class="TabDatenbank" name="tabDatenbank">
           <attribute name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/light/tabdatenbank.svg</normaloff>
             <disabledoff>:/images/disabled/tabdatenbank.svg</disabledoff>:/images/light/tabdatenbank.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Datenbank</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeuen_Sud_anlegen"/>
    <addaction name="actionSud_kopieren"/>
    <addaction name="actionSud_teilen"/>
    <addaction name="actionSud_l_schen"/>
    <addaction name="separator"/>
    <addaction name="actionRezept_importieren"/>
    <addaction name="actionRezept_exportieren"/>
    <addaction name="separator"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionVerwerfen"/>
    <addaction name="separator"/>
    <addaction name="actionDruckvorschau"/>
    <addaction name="actionDrucken"/>
    <addaction name="separator"/>
    <addaction name="actionBereinigen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <widget class="QMenu" name="menuFarbschema">
     <property name="title">
      <string>Farbschema</string>
     </property>
     <addaction name="actionThemeHell"/>
     <addaction name="actionThemeDunkel"/>
    </widget>
    <widget class="QMenu" name="menuStil">
     <property name="title">
      <string>Stil</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuReiter">
     <property name="title">
      <string>Reiter</string>
     </property>
     <addaction name="actionReiterGaerverlauf"/>
     <addaction name="actionReiterZusammenfassung"/>
     <addaction name="actionReiterEtikette"/>
     <addaction name="actionReiterBewertung"/>
     <addaction name="actionReiterBrauuebersicht"/>
     <addaction name="actionReiterAusruestung"/>
     <addaction name="separator"/>
     <addaction name="actionReiterDatenbank"/>
    </widget>
    <addaction name="actionWiederherstellen"/>
    <addaction name="separator"/>
    <addaction name="menuFarbschema"/>
    <addaction name="menuStil"/>
    <addaction name="actionSchriftart"/>
    <addaction name="separator"/>
    <addaction name="menuReiter"/>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionSpende"/>
    <addaction name="actionBierspende"/>
    <addaction name="separator"/>
    <addaction name="actionFormelsammlung"/>
    <addaction name="separator"/>
    <addaction name="actionUeber"/>
   </widget>
   <widget class="QMenu" name="menuSud">
    <property name="title">
     <string>Sud</string>
    </property>
    <addaction name="actionSudGebraut"/>
    <addaction name="actionSudAbgefuellt"/>
    <addaction name="actionSudVerbraucht"/>
    <addaction name="separator"/>
    <addaction name="actionHefeZugabeZuruecksetzen"/>
    <addaction name="actionWeitereZutaten"/>
    <addaction name="separator"/>
    <addaction name="actionEingabefelderEntsperren"/>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <widget class="QMenu" name="menuSprache">
     <property name="title">
      <string>Sprache</string>
     </property>
     <addaction name="actionDeutsch"/>
     <addaction name="actionEnglisch"/>
     <addaction name="actionSchwedisch"/>
     <addaction name="actionNiederlaendisch"/>
    </widget>
    <addaction name="actionOeffnen"/>
    <addaction name="separator"/>
    <addaction name="actionBestaetigungBeenden"/>
    <addaction name="actionCheckUpdate"/>
    <addaction name="actionTooltips"/>
    <addaction name="actionAnimationen"/>
    <addaction name="menuSprache"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSud"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuEinstellungen"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/save.svg</normaloff>
     <disabledoff>:/images/disabled/save.svg</disabledoff>:/images/light/save.svg</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="whatsThis">
    <string notr="true">save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVerwerfen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/restore.svg</normaloff>
     <disabledoff>:/images/disabled/restore.svg</disabledoff>:/images/light/restore.svg</iconset>
   </property>
   <property name="text">
    <string>Änderungen verwerfen</string>
   </property>
   <property name="whatsThis">
    <string notr="true">restore</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionWiederherstellen">
   <property name="text">
    <string>Zurücksetzen</string>
   </property>
  </action>
  <action name="actionReiterDatenbank">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Datenbank</string>
   </property>
  </action>
  <action name="actionUeber">
   <property name="text">
    <string>Über</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSpende">
   <property name="text">
    <string>Spende</string>
   </property>
  </action>
  <action name="actionSudGebraut">
   <property name="text">
    <string>&quot;Sud gebraut&quot; zurücksetzen</string>
   </property>
  </action>
  <action name="actionSudAbgefuellt">
   <property name="text">
    <string>&quot;Sud abgefüllt&quot; zurücksetzen</string>
   </property>
  </action>
  <action name="actionSudVerbraucht">
   <property name="text">
    <string>&quot;Sud verbraucht&quot; zurücksetzen</string>
   </property>
  </action>
  <action name="actionWeitereZutaten">
   <property name="text">
    <string>&quot;Zusätze Zugabe&quot; zurücksetzen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionOeffnen">
   <property name="text">
    <string>Datenbank auswählen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionThemeHell">
   <property name="text">
    <string>Hell</string>
   </property>
  </action>
  <action name="actionThemeDunkel">
   <property name="text">
    <string>Dunkel</string>
   </property>
  </action>
  <action name="actionBestaetigungBeenden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bestätigung beim Beenden</string>
   </property>
  </action>
  <action name="actionSchriftart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Systemschriftart</string>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auf Updates prüfen</string>
   </property>
  </action>
  <action name="actionHefeZugabeZuruecksetzen">
   <property name="text">
    <string>&quot;Hefe Zugabe&quot; zurücksetzen</string>
   </property>
  </action>
  <action name="actionBereinigen">
   <property name="text">
    <string>Bereinigen</string>
   </property>
  </action>
  <action name="actionEingabefelderEntsperren">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eingabefelder entsperren</string>
   </property>
  </action>
  <action name="actionTooltips">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tooltips</string>
   </property>
  </action>
  <action name="actionNeuen_Sud_anlegen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_neu.svg</normaloff>
     <disabledoff>:/images/disabled/sud_neu.svg</disabledoff>:/images/light/sud_neu.svg</iconset>
   </property>
   <property name="text">
    <string>Neuen Sud anlegen</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_neu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRezept_importieren">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_import.svg</normaloff>
     <disabledoff>:/images/disabled/sud_import.svg</disabledoff>:/images/light/sud_import.svg</iconset>
   </property>
   <property name="text">
    <string>Rezept importieren</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_import</string>
   </property>
  </action>
  <action name="actionRezept_exportieren">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_export.svg</normaloff>
     <disabledoff>:/images/disabled/sud_export.svg</disabledoff>:/images/light/sud_export.svg</iconset>
   </property>
   <property name="text">
    <string>Rezept exportieren</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_export</string>
   </property>
  </action>
  <action name="actionSud_kopieren">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_kopie.svg</normaloff>
     <disabledoff>:/images/disabled/sud_kopie.svg</disabledoff>:/images/light/sud_kopie.svg</iconset>
   </property>
   <property name="text">
    <string>Sud kopieren</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_kopie</string>
   </property>
  </action>
  <action name="actionSud_l_schen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_loeschen.svg</normaloff>
     <disabledoff>:/images/disabled/sud_loeschen.svg</disabledoff>:/images/light/sud_loeschen.svg</iconset>
   </property>
   <property name="text">
    <string>Sud löschen</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_loeschen</string>
   </property>
  </action>
  <action name="actionSud_teilen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/sud_teilen.svg</normaloff>
     <disabledoff>:/images/disabled/sud_teilen.svg</disabledoff>:/images/light/sud_teilen.svg</iconset>
   </property>
   <property name="text">
    <string>Sud teilen</string>
   </property>
   <property name="whatsThis">
    <string notr="true">sud_teilen</string>
   </property>
  </action>
  <action name="actionDruckvorschau">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/print.svg</normaloff>
     <disabledoff>:/images/disabled/print.svg</disabledoff>:/images/light/print.svg</iconset>
   </property>
   <property name="text">
    <string>Drucken</string>
   </property>
   <property name="whatsThis">
    <string notr="true">print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/light/pdf.svg</normaloff>
     <disabledoff>:/images/disabled/pdf.svg</disabledoff>:/images/light/pdf.svg</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="whatsThis">
    <string notr="true">pdf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionReiterEtikette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Etikette</string>
   </property>
  </action>
  <action name="actionReiterBewertung">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bewertung</string>
   </property>
  </action>
  <action name="actionReiterZusammenfassung">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spickzettel/Zusammenfassung</string>
   </property>
  </action>
  <action name="actionReiterBrauuebersicht">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Brauübersicht</string>
   </property>
  </action>
  <action name="actionAnimationen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animationen</string>
   </property>
  </action>
  <action name="actionBierspende">
   <property name="text">
    <string>Bierspende</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/de.svg</normaloff>:/images/de.svg</iconset>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actionEnglisch">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/en.svg</normaloff>:/images/en.svg</iconset>
   </property>
   <property name="text">
    <string>Englisch</string>
   </property>
  </action>
  <action name="actionSchwedisch">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/se.svg</normaloff>:/images/se.svg</iconset>
   </property>
   <property name="text">
    <string>Schwedisch</string>
   </property>
  </action>
  <action name="actionNiederlaendisch">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/nl.svg</normaloff>:/images/nl.svg</iconset>
   </property>
   <property name="text">
    <string>Niederländisch</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionFormelsammlung">
   <property name="text">
    <string>Formelsammlung</string>
   </property>
  </action>
  <action name="actionReiterAusruestung">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ausrüstung</string>
   </property>
  </action>
  <action name="actionReiterGaerverlauf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gärverlauf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TabRohstoffe</class>
   <extends>QWidget</extends>
   <header>tabrohstoffe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabSudAuswahl</class>
   <extends>QWidget</extends>
   <header>tabsudauswahl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabAusruestung</class>
   <extends>QWidget</extends>
   <header>tabausruestung.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabRezept</class>
   <extends>QWidget</extends>
   <header>tabrezept.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabBrauUebersicht</class>
   <extends>QWidget</extends>
   <header>tabbrauuebersicht.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabBraudaten</class>
   <extends>QWidget</extends>
   <header>tabbraudaten.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabGaerverlauf</class>
   <extends>QWidget</extends>
   <header>tabgaerverlauf.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabZusammenfassung</class>
   <extends>QWidget</extends>
   <header>tabzusammenfassung.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabEtikette</class>
   <extends>QWidget</extends>
   <header>tabetikette.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabBewertung</class>
   <extends>QWidget</extends>
   <header>tabbewertung.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabDatenbank</class>
   <extends>QWidget</extends>
   <header>tabdatenbank.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabAbfuellen</class>
   <extends>QWidget</extends>
   <header>tababfuellen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
